gpt4all-chat/qml/Theme.qml

```diff
@@ -692,9 +707,9 @@ QtObject {
     property color welcomeButtonText: {
         switch (MySettings.chatTheme) {
             case "LegacyDark":
-                return textColor
+                return textColor;
```

Comment:
Technically, the semi-colon is not necessary. I'd take a change that just removed all semicolons.